packages/fx-core/src/common/samples-config.json

```diff
@@ -90,7 +90,7 @@
             "fullDescription": "Todo List app helps to manage personal to do items and also runs across Microsoft 365, including Teams, Outlook and Office. The frontend is a React app and the backend is an Azure Function. You can optionally deploy and host the app in Azure.",
             "tags": ["React", "Azure function", "JS", "Outlook", "Office"],
             "time": "5min to run",
-            "configuration": "F5 ready"
+            "configuration": "Ready for local debug"
```

Comment:
This sample is not local debug targeted. The goal of this sample is to run the todo list app in Outlook and Office, which needs provision & deploy. Currently, local debug only works in Teams, but not in Outlook and Office. So I think `Ready for local debug` may be not very suitable. @MuyangAmigo @zyxiaoyuer What do you think about?

Reply:
Updated